resources/keywords/create_instance.resource

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Resource    ../keywords/deploy_instance_dev.resource
+Resource    ../object_repo/create_dbinstance_obj.resource
+Resource    ../object_repo/database_access_obj.resource
+Library     ../../utils/scripts/util.py
+
+*** Variables ***
+${instanceName}      ${EMPTY}
```

Comment:
../keywords/deploy_instance_dev.resource already has this variable, we can just call it out instead of declaring it.